README.md

```diff
@@ -25,6 +25,7 @@ It creates a task 'nunit' that may be configured as follows:
 
     nunit {
         // optional - defaults to '2.6.4', but plugin is compatible with v3+ as well
+        // it must be the *first* parameter if specified
```

Comment:
I agree
Actually it is only required when you specific version specific settings and I think it is valid reason for that
I could update the comment to state it clearer